### DISCUSSION TITLE
REST API for MinIO

### DISCUSSION THREAD

**Discussion by SriRamJai8** [minio/minio#20236] [Q&A]
Do we have REST API for MInIO, and may be postman collections ? I am trying to run few api calls to get bucket info etc.

**Comment by marktheunissen** [comment]
MinIO is compatible with S3 clients, and we have our own SDKs for many popular languages, and tools for making API calls, see:

- https://github.com/minio/minio-go
- https://github.com/minio/mc

etc.
